core/selection.js

```diff
@@ -75,10 +76,12 @@ class Selection {
   }
 
   handleDragging() {
-    this.emitter.listenDOM('mousedown', document.body, () => {
+    const node = (this.rootDocument === document ? document.body : this.rootNode);
+
+    this.emitter.listenDOM('mousedown', node, () => {
```

Comment:
should these be reverted to `document`? why do we listen on document for mouse events? shouldn't we only listen on the editor container?

Reply:
We want to know if you drag outside of the boundaries of the editor container. See the lines below this one.

Reply:
ah fair enough, this can probably be reverted then as its specific to the body rather than the container.